native/cocos/platform/interfaces/modules/canvas/CanvasRenderingContext2D.cpp

```diff
@@ -307,10 +309,10 @@ void CanvasRenderingContext2D::setFont(const ccstd::string &font) {
         }
 
         double fontSize = atof(fontSizeStr.c_str());
-        bool isBold = font.find("bold", 0) != ccstd::string::npos;
-        bool isItalic = font.find("italic", 0) != ccstd::string::npos;
-        bool isSmallCaps = font.find("small-caps", 0) != ccstd::string::npos;
-        bool isOblique = font.find("oblique", 0) != ccstd::string::npos;
+        bool isBold = font.find("bold", 0) != ccstd::string::npos || font.find("Bold", 0) != ccstd::string::npos;
+        bool isItalic = font.find("italic", 0) != ccstd::string::npos || font.find("Italic", 0) != ccstd::string::npos;
```

Comment:
Can use the same logic for all platforms?

Reply:
Yes. because resouce name is same and all platforms use the name to set parameters.  Have tested on ios/mac/android/win.